NetCoreRestApi/DataLayer/Converters/CategoryEntityConverter.cs

```diff
@@ -0,0 +1,39 @@
+﻿using DataLayer.Models;
+using Common.Converter;
+using DataLayer.Entities;
+
+namespace DataLayer.Converters
+{
+    public class CategoryEntityConverter : IConverter<CategoryModel, CategoryEntity>
```

Comment:
1. Rename to CategoryModelConverter
2. Swap in the interface

Reply:
done